### PR TITLE
Add fee estimate to payments page

### LINKED ISSUE #1080: Add fee estimate for LN payments

### PR DESCRIPTION
**Pull request by niteshbalusu11**
Closes #1080 

Add fee estimate to payments page.

### CODE REVIEW

**Review by djkazic — APPROVED** [review]

File: src/windows/Send/SendConfirmation.tsx [hunk 1]
```diff
@@ -62,6 +81,35 @@ export default function SendConfirmation({ navigation, route }: ISendConfirmatio
...
+                  }
+                }
+                } catch (error) {
+                console.log(error);
+            }
+          }
+        }
+
+        // Delay the execution of the function 1 second so that we don't query too quickly.
+        timeoutId = setTimeout(getFeeEstimate, 1000);
```
Comment (hsjoberg) [inline src/windows/Send/SendConfirmation.tsx:107]: I think this should use the debouncer instead. Check out how it's done in ReceiveSetup.tsx

**Review by hsjoberg — CHANGES_REQUESTED** [review]
Looks good! But use the debouncer instead.

File: src/windows/Send/SendConfirmation.tsx (hunk 1 above)
Comment (niteshbalusu11) [reply src/windows/Send/SendConfirmation.tsx]: Done

**Comment by niteshbalusu11** [comment]
Please check now.

**Review by hsjoberg — APPROVED** [review]

**Pull request merged**